draft-ietf-tls-dtls-connection-id.md

```diff
@@ -250,13 +250,13 @@ zeros
          uint48 sequence_number;
          opaque cid[cid_length];               // New field
          uint16 length;
-         opaque enc_content[DTLSCiphertext.length];
-     } DTLSCiphertext;
+         opaque enc_content[ciphertext_length];
+     } DTLSCIDCiphertext;
```

Comment:
I may be wrong.
For me, the old defines a `DTLSCiphertext`, which contains a `length` field, which is referenced by `DTLSCiphertext.length`. the new one now has still that field, but introduces a "ciphertext_length",  which may be considered as "maverick".

Reply:
Would `DTLSCIDCiphertext.length` be a replacement for "ciphertext_length"?

Reply:
Using DTLSCIDCiphertext.length is IMHO incorrect because the length of the encrypted content is different from the length of the entire DTLSCIDCiphertext structure. Do I see that wrong?

Reply:
My understanding:
`DTLSCIDCiphertext.length` is not the length of `DTLSCIDCiphertext`, it's the value of the field `length` of that structure.

Reply:
Yes, this interpretation is correct.